site/src/Map.jsx

```diff
@@ -1,135 +1,238 @@
           onClick={onClick}
           cursor={cursor}
           hash={true}
+          mapStyle={MAP_STYLE}
           interactiveLayerIds={interactiveLayerIds}
           initialViewState={INITIAL_VIEW_STATE}
-          mapStyle={getMapStyle()}
-          style={{position: 'fixed', width: '100%', height: 'calc(100vh - 60px)'}}
+          style={{
+            position: "fixed",
+            width: "100%",
+            height: "calc(100vh - 60px)",
+          }}
+          attributionControl={false}
         >
-          <Source id="overture-places" type="vector" url={PLACES_PMTILES_URL}>
-            <Layer {...PLACES_MAP_STYLE} layout={{visibility: interactiveLayerIds.includes('places') ? 'visible' : 'none'}} />
-          </Source>
-          <NavigationControl position='top-right'></NavigationControl>
+          <ThemeSource name="buildings" url={PMTILES_URL} />
+          <ThemeSource name="places" url={PMTILES_URL} />
+          <ThemeSource name="divisions" url={PMTILES_URL} />
+
+          <Layer
+            id="divisions_division"
+            type="symbol"
+            source="divisions"
+            source-layer="division"
+            paint={{
+              "text-color": mode === "theme-light" ? "black" : "white",
+              "text-halo-color": mode === "theme-light" ? "white" : "black",
+              "text-halo-width": 1,
+            }}
+            layout={{
+              "text-font": ["Noto Sans Regular"],
+              "text-field": ["get", "@name"],
+              "text-size": 11,
+            }}
+          />
+
+          {visibleThemes.includes("divisions") ? (
+            <>
+              <ThemeTypeLayer
+                theme="divisions"
+                type="division_area"
+                polygon
+                color="#bc80bd"
+              />
+              <ThemeTypeLayer
+                theme="divisions"
+                type="boundary"
+                line
+                color="#bc80bd"
+              />
+            </>
+          ) : null}
+          {visibleThemes.includes("buildings") ? (
+            <>
+              <ThemeTypeLayer
+                theme="buildings"
+                type="building"
+                polygon
+                color="#d9d9d9"
+              />
+              <ThemeTypeLayer
+                theme="buildings"
+                type="building_part"
+                polygon
+                color="#d9d9d9"
+              />
+            </>
+          ) : null}
+          {visibleThemes.includes("places") ? (
+            <ThemeTypeLayer theme="places" type="place" point color="#fdb462" />
+          ) : null}
+
+          <NavigationControl position="top-right"></NavigationControl>
           <GeolocateControl />
+          <AttributionControl customAttribution='<a href="https://openstreetmap.org/copyright" target="_blank">© OpenStreetMap contributors</a>, <a href="https://overturemaps.org" target="_blank">Overture Maps Foundation</a>' />
```

Comment:
Unfortunately `customAttribution` cannot be reactive.

Reply:
Let's add an AI to discuss attribution during the next TF meeting. I think we'll need guidance on the exact text for each layer.

Open questions I have are:
1. Any other ODBL licensed data we need to include explicitly in the attribution?
2. What does attribution look like for Places?
3. Is there a better link for Overture attribution? Seems like a more robust release manifest which includes all the sources for each theme would be more appropriate. Might not exist now, but worth starting the discussion.

Reply:
We should model appropriate attribution behavior before go-live, agreed!

Reply:
We can also augment the global attribution with embedded attribution within the inspector panel. For example, and active entity which was derived from OSM contains a link to the entity on OSM Website.